build.gradle

```diff
@@ -1,29 +1,22 @@
-buildscript {
-    dependencies {
-        classpath("net.serenity-bdd:serenity-gradle-plugin:2.6.0")
-        classpath("net.serenity-bdd:serenity-single-page-report:2.6.0")
-    }
-}
-
 plugins {
     id 'application'
     id 'uk.gov.hmcts.java' version '0.12.43'
     id 'io.spring.dependency-management' version '1.1.0'
-    id 'org.springframework.boot' version '2.7.12'
+    id 'org.springframework.boot' version '3.1.0'
     id 'com.github.ben-manes.versions' version '0.47.0'
     id 'org.sonarqube' version '4.2.1.3168'
     id 'jacoco'
     id 'checkstyle'
     id 'io.freefair.lombok' version '8.1.0'
     id 'info.solidsoft.pitest' version '1.9.0'
     id 'org.owasp.dependencycheck' version '8.3.1'
-    id 'au.com.dius.pact' version '4.2.14'
+    id 'au.com.dius.pact' version '4.6.0'
```

Comment:
Updating this will break the master build. Please refer to this as example : https://github.com/hmcts/em-annotation-api/pull/1406/files
And revert the 2 libraries to the versions in referenced PR.